### PR TITLE
Add Readme and some tests to :erc961

### PR DESCRIPTION
**Pull request by bigman212**

### CODE REVIEW

File: erc831/src/main/kotlin/org/kethereum/erc831/ERC831.kt [hunk 1]
```diff
@@ -1,9 +1,10 @@
 package org.kethereum.erc831
 
-// as defined in http://eips.ethereum.org/EIPS/eip-831
-
+/**
+ * as defined in [EIP-831](https://eips.ethereum.org/EIPS/eip-831)
```
Comment (bigman212) [inline erc831/src/main/kotlin/org/kethereum/erc831/ERC831.kt:4]: do I understand that 831 is not yet approved ERC, but still EIP?

File: erc831/src/main/kotlin/org/kethereum/erc831/ERC831Parser.kt [hunk 2]
```diff
@@ -39,12 +39,13 @@ fun EthereumURI.toERC831() = ERC831().apply {
...
 
 }
 
-private fun String.hasPrefix() = contains('-') && (!contains("0x") || indexOf('-') < indexOf("0x"))
+private fun String.hasPrefix(): Boolean =
+    contains('-') && (!contains("0x") || indexOf('-') < indexOf("0x"))
 
-fun parseERC831(url: String) = EthereumURI(url).toERC831()
\ No newline at end of file
+fun parseERC831(url: String): ERC831 = EthereumURI(url).toERC831()
```
Comment (bigman212) [inline erc831/src/main/kotlin/org/kethereum/erc831/ERC831Parser.kt:51]: just explicit types and some wraps that are added by the Intellij

**Comment by bigman212** [comment]
@ligi ❤️

File: erc831/src/main/kotlin/org/kethereum/erc831/ERC831.kt (hunk 1 above)
Comment (ligi) [reply erc831/src/main/kotlin/org/kethereum/erc831/ERC831.kt:4]: yea - still need to push this through the EIP process

**Review by ligi — APPROVED** [review]
LGTM - Thanks and sorry for the late merge

**Pull request merged**